src/_pyfuse3.py

```diff
@@ -65,7 +65,7 @@ class Operations:
     enable_writeback_cache: bool = False
     enable_acl: bool = False
 
-    def init(self) -> None:
+    def init(self, conn_info) -> None:
         '''Initialize operations.
 
         This method will be called just before the file system starts handling
```

Comment:
Please document the parameter and its use.

Reply:
done, best English I could synthesize ;)

Reply:
Um, I don't see any documentation here?